src/addons/zigbee-adapter/zb-adapter.js

```diff
@@ -261,7 +261,7 @@ class ZigBeeAdapter extends Adapter {
       // We're going to change something, so we might as well set the link
       // key, since it's write only and we can't tell if its been set before.
       configCommands.push(this.AT(AT_CMD.LINK_KEY,
-                                  {linkKey: 'ZigBeeAlliance09'}));
+                                  {linkKey: 'ZigbeeAlliance09'}));
```

Comment:
I think this one maybe should have stayed `ZigBeeAlliance09`

Reply:
Argh. You are correct. (hangs head in shame).